packages/unity-react-core/src/components/PersonProfile/PersonProfile.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { PersonProfile, SocialMediaPlatform } from "./PersonProfile";
+import img from "../../assets/img/anon.png";
```

Comment:
Can we try this out using the image in the shared folder: 
Replace: `"../../assets/img/anon.png" ;`
with: `"@shared/assets/img/named/anon.png";`
I can't remember if the `@shared` alias is working, use the long path if you have to
with: `"../../../../../shared/assets/img/named/anon.png";`